recipes/libssh2/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mkdir build && cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX
+
+cmake --build . --config Release --target install
+
+if [ -d $PREFIX/lib64 ];then
+    mv $PREFIX/lib64 $PREFIX/lib
```

Comment:
Already exists, because conda-build creates it? So, it's guaranteed to exist and no need for something like this?

`mkdir $PREFIX/lib`